concrete/src/Page/Page.php

```diff
@@ -3690,6 +3696,63 @@ public function getCollectionFullPageCachingLifetimeValue()
             $lifetime = 31536000;
         }
 
+        /** @var Date $date */
+        $date = $app->make(Date::class);
+        $now = $date->getOverridableNow();
+        /** @var Connection $connection */
+        $connection = $app->make(Connection::class);
+
+        // Get upcoming publish date
+        $upcomingPublishDate = $connection->createQueryBuilder()
+            ->select('cv.cvPublishDate')
+            ->from('collectionVersions', 'cv')
```

Comment:
We should have caught this but this breaks on all case sensitive systems (which is most of them.)

Reply:
Oops. Thanks.